posthog/temporal/data_modeling/run_workflow.py

```diff
@@ -382,7 +382,7 @@ def hogql_table(query: str, team: Team, table_name: str, table_columns: dlt_typi
     """A dlt source representing a HogQL table given by a HogQL query."""
 
     async def get_hogql_rows():
-        settings = HogQLGlobalSettings(max_execution_time=60 * 10)  # 10 mins, same as the /query endpoint async workers
+        settings = HogQLGlobalSettings(max_execution_time=60 * 20)  # 20 mins, 2x times the /query endpoint async workers
```

Comment:
I assume that the query that's timing out is the one running with these settings. Then why are we also updating the `HOGQL_INCREASED_MAX_EXECUTION_TIME`? Isn't it kind of redundant to both bump that setting and also set `max_execution_time` here?

Maybe there is something I'm missing as I'm not super familiar with HogQL.

Reply:
I am not as well, but I think you're right and we should just override this one.